main.js

```diff
@@ -54,19 +54,20 @@ module.exports = function(options) {
 	app.locals.__rootDirectory = directory;
 
 
-	app.use('/' + name, express.static(directory + '/public', {
-		setHeaders: function(res) {
-			// TODO:MA Once we are generating new paths on every deploy (git hash?) then up the max-age to 'a long time'
-			res.setHeader('Cache-Control', 'max-age=120, public, stale-while-revalidate=259200, stale-if-error=259200');
-		}
-	}));
+	if (!app.locals.__isProduction) {
```

Comment:
@ironsidevsquincy I would argue this is a feature not a breaking API change :wink: